### DISCUSSION TITLE
Migration scraping and TSDB to another server

### DISCUSSION THREAD

**Discussion by Mojito2110** [prometheus/prometheus#12537] [General]
Hello,

is it possible to implement the migration scenario below :
move the TSDB from a Prometheus 2.2x or 2.3x server to a Prometheus 2.4x server

Despite the version difference, is the move possible? 
What are the steps involved in switching from one server to another while limiting data loss? I'm having trouble finding the details of the operations to be carried out for this type of scenario (stopping the servers, copying, etc.... ). 

Thanks for your help

**Comment by roidelapluie** [comment]
You would need to stop the server and copy the data over yet. Within 2.x, it will work.
